test/marketsDistribution.test.ts

```diff
@@ -11,4 +11,8 @@ describe("Markets distribution", () => {
     const marketDistribution = await getMarketsDistribution(now(), provider);
     expect(marketDistribution).not.toBeUndefined();
   });
+  it("Should now include the snapshotProposal field (true for every epoch starting at age3)", async () => {
+    const marketDistribution = await getMarketsDistribution();
```

Comment:
Use a static timestamp to not make the test failing in the future. We cannot be sure today that the DAO will always use Snapshot votes mechanism for future ages.

Reply:
if we deprecate snapshot we shouln't keep a test checking we use snapshot.

Reply:
I would rather delete the test

Reply:
but we want to keep the things working on the past epochs
So IMO it is a good thing to keep the test for the epoch with a snapshot proposal available